scripts/binddojo.bat

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Aren't you able to execute shell scripts on windows? I would prefer not to add this, since it isn't maintainable for most contributors

Reply:
Did not see this until now. Getting to my pc

Reply:
You can execute shell scripts on Windows, however the escape pathing just doesn't work natively, so none of the files will be found.  Sorry for the late reply